src/interactive/Tooltip.jsx

```diff
@@ -167,9 +176,10 @@ Tooltip.propTypes = {
 	trigger: PropTypes.element.isRequired,
 	content: PropTypes.element,
 	align: PropTypes.oneOf(["left", "right", "center"]).isRequired,
+	popAbove: PropTypes.bool,
```

Comment:
Is this prop name intuitive? Should we do something more flexible like `popupFrom=[left|right|top|bottom]`?

Reply:
My comment above is related, but starting a new prop and treating `align` as deprecated could also be an option. Maybe `position=[left|right|top|bottom]`?

Reply:
A separate prop allows users to control horizontal and vertical alignment separately. `popAbove` allows someone position the tooltip using `dropdown-content--above dropdown-content--right`. It depends on if we think we need that level of control, or if `[left|right|top|bottom]` is enough